### PR TITLE
Rename Cardano.Wallet.StakePool.Metrics to Cardano.Pool.Metrics

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

#711 

# Overview

- [x] I have renamed `Cardano.Wallet.StakePool.Metrics` to `Cardano.Pool.Metrics`

# Comments

### CODE REVIEW

File: lib/core/cardano-wallet-core.cabal [hunk 1]
```diff
@@ -116,7 +116,7 @@ library
       Cardano.Wallet.Primitive.Model
       Cardano.Wallet.Primitive.Types
       Cardano.Wallet.Registry
-      Cardano.Wallet.StakePool.Metrics
+      Cardano.Pool.Metrics
```
Comment (KtorZ) [inline lib/core/cardano-wallet-core.cabal]: May we keep this list sorted please :) ?

**Review by KtorZ — APPROVED** [review]

**Review by paweljakubas — APPROVED** [review]
needed code reshufling

**Pull request merged**